### DISCUSSION TITLE
优化pag文件体积大小的问题

### DISCUSSION THREAD

**Discussion by doruei** [Tencent/libpag#2460] [General] [answered]
在优化指南(https://pag.io/docs/optimization.html)的基础上进行测试,发现有几个问题,比较困惑，麻烦解答一些，谢谢:
前提：全BMP导出，无矢量或者混合导出
1、将ae动画导出为帧序列png后，压缩后拉回ae进行BMP导出，发现压缩后的pag文件体积反而大于未压缩的，请问这是正常的吗？
2、直接缩小合成分辨率（未缩小素材分辨率），比如1170缩小成700，体积从1.5m下降成1.4m，但如果缩小成300，体积则变成200k，为啥会出现这个情况？
3、文档中有写到降低素材分辨率能降低pag文件体积，但帧序列图片尺寸从1170降低至750，导出pag后体积没有太大变化，

**Comment by domchen** [comment, marked as answer]
1.PAG插件已经拥有最高的压缩率，不要试图提前压缩，你做的任何对原始素材的提前压缩都是损失清晰度，而且可能造成文件更大。一律使用最高清素材给PAG插件导出即可。这点在FAQ里也写了。
2和3是相同的问题。有一条隐藏规则是BMP预合成存在导出分辨率上限，默认值是短边长不超过720。具体可以在插件选项面板里设置修改。所以你改到700附近的分辨率变化不大，因为之前也是导出的最大就720P。
